src/JustEat.StatsD/JustEat.StatsD.csproj

```diff
@@ -13,6 +13,7 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'net451' ">
     <Reference Include="System" />
     <Reference Include="System.Runtime" />
+    <Reference Include="System.Xml" />
```

Comment:
wat?

Reply:
This is an old commit, I was testing using an actual `System.Xml` type with a workaround to see if I could prevent the potential issue, which I did in a later commit and reverted it cause I couldn't get it working.